code/game/machinery/atmoalter/canister.dm

```diff
@@ -475,6 +492,11 @@ update_flag
 	src.air_contents.adjust_gas(GAS_PHORON, MolesForPressure())
 	src.update_icon()
 
+/obj/machinery/portable_atmospherics/canister/hydrogen/engine_setup/Initialize()
```

Comment:
This is identical to its parent, which you also define.

Reply:
Didn't actually think about that when I was shoving the code over - It looks like all of the engine_setup variants are literally the same as their parents, the only difference is that they are used in the setup_supermatter.dm file, and nowhere else? I'll just make the setup file use the parent canisters and delete the engine_setup variants.